src/intTest/kotlin/no/nav/personbruker/dittnav/eventaggregator/informasjon/InformasjonQueriesTest.kt

```diff
@@ -46,14 +46,6 @@ class InformasjonQueriesTest {
         }
     }
 
-    @AfterAll
```

Comment:
Hvorfor er denne fjernet? Vil ikke det kunne skape støy for andre tester at det ikke ryddes opp her?